controllers/namespace_controller.go

```diff
@@ -76,16 +77,30 @@ func (r *NamespaceReconciler) reconcile(ctx context.Context, ns *corev1.Namespac
 func (r *NamespaceReconciler) propagateMeta(ctx context.Context, ns, parent *corev1.Namespace) error {
 	orig := ns.DeepCopy()
 	for _, key := range r.LabelKeys {
-		if val, ok := parent.Labels[key]; ok {
-			ns.Labels[key] = val
+		for label, val := range parent.Labels {
+			ok, err := filepath.Match(key, label)
+			if err != nil {
```

Comment:
The only possible error here is `filepath.ErrBadPattern` here,
and such a bad pattern can be eliminated beforehand when
loaded from the configuration file.

Please validate the pattern when loaded.

Reply:
Thank you comments.
I fixed:
https://github.com/cybozu-go/accurate/pull/8/commits/88a1a9a6e7dbf4dbf805057a465ea56732d6942e